### ISSUE TITLE
Train GaitPart on OU-MVLP

### ISSUE THREAD

**Issue by w1998w** [ShiqiYu/OpenGait#14]
您好，感谢您出色的工作。
我训练GaitPart'在CASIA-B数据集上的训练结果跟您训练结果相近，但是在训练OU-MVLP时，在一块1080TI和GTX TITAN上训练，batch_size设为（24，16），训练350k iteration后准确率才刚79%左右不到80%。
请问您项目的代码修改配置文件中关于数据集的路径之后可以直接对OUMVLP数据集进行训练而不需要再对某些代码部分进行修改了吗？

**Comment by silverlining21** [comment]
![Uploading image.png…]()


modify the channel setting, hope helps

**Comment by w1998w** [comment]
> modify the channel setting, hope helps

不好意思，我这边无法显示你上传的图片。论文中除了这个地方
In OU-MVLP , due to it contains almost 20 times more sequences than CASIA-B, an additional block composed of two FConv Layers is stacked into the FPFE (the output channel is set to 256) and the value of p of each block is set to 1, 1, 3, 3 respectively. The batch size is set to (32, 16), the iterations is set to 250K, and the learning rate would be reduced to 1e-5 at 150k iterations.
在训练两个数据集的时候应该都是一样的吧，所以网络只需要改这个地方就可以了吧？
感谢你的帮助

**Comment by silverlining21** [comment]
pls check `Table 1. Network structures of the proposed method` of the orignal papaer detaily.

BRs
